testsuite/features/init_clients/proxy_container.feature

```diff
@@ -29,10 +29,10 @@ Feature: Setup containerized proxy
     And I click on "Bootstrap"
     And I wait until I see "Bootstrap process initiated." text
 
-  # workaround for bsc#1218146
-  Scenario: Reboot the proxy host
-    When I reboot the "proxy" minion through SSH
-    And I wait until port "22" is listening on "proxy" host
+#  workaround for bsc#1218146, once we start using SLE/Leap Micro https://github.com/SUSE/spacewalk/issues/23811
+#  Scenario: Reboot the proxy host
+#    When I reboot the "proxy" minion through SSH
+#    And I wait until port "22" is listening on "proxy" host
```

Comment:
Same here. In the HEAD CI, the default image for proxy is SLE Micro 5.5. See https://github.com/uyuni-project/sumaform/blob/90ef843cc7a62184328bfe23d13d4393c5957480/modules/proxy_containerized/main.tf#L3